test/integration/test_upgrade.py

```diff
@@ -49,6 +51,17 @@ def testSkipunzip(self):
 
     @pytest.mark.slowtest
     def testUpgrade(self):
+        args = ["--branch=OMERO-DEV-latest"]
+        # Python 3.6 on Travis: Force OMERO to run with 2.7 instead
+        if getenv('TRAVIS_PYTHON_VERSION') == '3.6':
+            args += ['--python', 'python2.7']
+        self.upgrade(*args)
+
+    @pytest.mark.slowtest
+    @pytest.mark.skipif(
+        getenv('TRAVIS_PYTHON_VERSION') == '3.6',
+        reason='OMERO not supported on Python 3.6')
```

Comment:
worth a todo somewhere for bumping this?

Reply:
https://github.com/ome/omego/issues/125